README.md

```diff
@@ -1,2 +1,56 @@
-# cat-bnb
-Getting your cat the attention it deserves
+# Project Title
+
+## **Cat-BNB** 
+**Getting your cat the attention it deserves!**
+
+After reviewing the Swedish cyberspace, we discovered that there is no affordable solution for cat owners that want to leave their cats to a safe place, when going on vacation.
+
+During our studies in Craft Academy, we decided to make a small basic application which covers that need.
+
+Inspired by AirBNB, we set out on May 2019 to create something similar, where interested cat hosts can post their details, for cat owners to find them.
+
+This was a one week project (May 6th - 12th).
+
+# Deployment
+
+We have deployed this application using Heroku. You can navigate through it......(to be announced).
+
+# Tests, test coverage & CI
+
+This application was feature tested using [Cucumber](https://cucumber.io/) and unit tested using [Rspec](https://rspec.info/).
+
+To be able to run the tests, run `bundle install` in your terminal as soon as you fork this repository.
+
```

Comment:
should we add here `rails db:migrate` ? (i am not sure so feel free to disagree)

Reply:
Right again. Added ;)